Directory.Build.props

```diff
@@ -335,6 +335,9 @@
     <AndroidNamespaceReplacement Include='viewtree' Replacement='ViewTree' />
     <AndroidNamespaceReplacement Include='restorecredential' Replacement='RestoreCredential' />
     <AndroidNamespaceReplacement Include='identitycredentials' Replacement='IdentityCredentials' />
+    <AndroidNamespaceReplacement Include='vobsub' Replacement='VobSub' />
```

Comment:
While I personally would have gone with `VobSub`, it appears that Google is going with `Vobsub`, so we should remove this line.

https://developer.android.com/reference/androidx/media3/extractor/text/vobsub/VobsubParser?hl=en